llvm/lib/Target/EraVM/EraVMInstrInfo.td

```diff
@@ -1214,15 +1217,15 @@ let hasSideEffects = 1 in {
 def L1r : ILogRrr_<OpLogToL1, (outs), (ins GR256:$rs0, GR256:$rs1),
```

Comment:
@asl `r` suffixes can be dropped from `def` names of a few instructions here, I guess (such as `L1Initialr -> L1Initial` or `PCOMPr -> PCOMP`). There is only one variant of each instruction and the suffixes should be `rr` or `rrr` anyway.

Reply:
(this can be updated later, of course)

Reply:
Created a separate follow-up PR to move this discussion there: #583.